docs/data/osdf/install-cache-container.md

```diff
@@ -0,0 +1,347 @@
+| `WEB_INTERFACE_PORT`  | `8444`                           | The host port used for service management via the web interface                         | 
+| `ISSUER_JWK`          | `/etc/pelican/issuer.jwk`        | The private key generated by `pelican generate keygen`                                  | 
+| `HOST_CERT_CHAIN`     | `/etc/pelican/hostcert.pem`      | The host certificate, with full chain                                                   | 
+| `HOST_KEY`            | `/etc/pelican/hostkey.pem`       | The private key matching the host certificate                                           | 
+| `CACHE_CONFIG_FILE`   | `/etc/pelican/cache-config.yaml` | The YAML file containing various cache settings                                         | 
+| `CACHE_PARTITION`     | `/mnt/cache`                     | The partition to be used for storing cached data (single partition setup only)          | 
+| `NAMESPACE_PARTITION` | `/mnt/cache-namespace`           | The partition to be used for storing namespace information (multi-partition setup only) | 
+| `METADATA_PARTITION`  | `/mnt/cache-meta`                | The partition to be used for storing metadata (multi-partition setup only)              | 
+| `DATA_PARTITION`      | `/mnt/cache-data`                | The partition to be used for storing cached file contents (multi-partition setup only)  | 
+
+
+### Single host partition
+
+A cache may be configured to use a single host partition only;
+this is a simpler setup than using multiple host partitions, but may have worse performance,
+and is not recommended for caches with more than 10 TB of capacity.
+
+
+```console
+user@host $ docker run --rm \
+             --publish <CACHE_HOST_PORT>:8443 \
+             --publish <WEB_INTERFACE_HOST PORT>:8444 \
+             --volume <ISSUER_JWK>:/etc/pelican/issuer.jwk \
+             --volume <HOST_CERT_CHAIN>:/etc/pelican/certificates/tls.crt \
+             --volume <HOST_KEY>:/etc/pelican/certificates/tls.key \
+             --volume <CACHE_CONFIG_FILE>:/etc/pelican/config.d/99-local.yaml \
+             --volume <CACHE_PARTITION>:/run/pelican/cache \
+             --name osdf-cache \
+             hub.opensciencegrid.org/pelican_platform/osdf-cache:latest
+```
+
+Using the example values from the table, this is
+
+```console
+user@host $ docker run --rm \
+             --publish 8443:8443 \
+             --publish 8444:8444 \
+             --volume /etc/pelican/issuer.jwk:/etc/pelican/issuer.jwk \
+             --volume /etc/pelican/hostcert.pem:/etc/pelican/certificates/tls.crt \
+             --volume /etc/pelican/hostkey.pem:/etc/pelican/certificates/tls.key \
+             --volume /etc/pelican/cache-config.yaml:/etc/pelican/config.d/99-local.yaml \
+             --volume /mnt/cache:/run/pelican/cache \
+             --name osdf-cache \
+             hub.opensciencegrid.org/pelican_platform/osdf-cache:latest
+```
+
+
+### Multiple host partitions
```

Comment:
Multiple "host partitions" or "multiple host" partitions?

Reply:
I just got rid of the word "host" since it's superfluous.